src/NServiceBus.Core.Analyzer/SymbolExtensions.cs

```diff
@@ -0,0 +1,30 @@
+﻿namespace NServiceBus.Core.Analyzer
+{
+    using System.Collections.Generic;
+    using Microsoft.CodeAnalysis;
+
+    static class SymbolExtensions
+    {
+        public static string GetFullName(this IMethodSymbol method)
```

Comment:
Is this the only way to get the full name of the method? This is rather allocation-heavy.